src/se/kth/inda17/style.css

```diff
@@ -0,0 +1,6 @@
+#pane {
+    -fx-background-image: url("background.png");
+    -fx-background-repeat: stretch;
+    -fx-background-size: 300 275;
```

Comment:
Would it be possible to automatically fill the window with the background image?